actionpack/lib/action_controller/metal/feature_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ActionController #:nodoc:
+  module FeaturePolicy
```

Comment:
Good spot for docs here (https://github.com/rails/rails/blob/6e40b131d2c7208e116d267e4b40dcf621f8b442/actionpack/lib/action_controller/metal/content_security_policy.rb#L5 notwithstanding 😅). Using the changelog entry verbatim could work well!

Reply:
Thanks for the pointer @jeremy! I wasn't sure if the intention was for code documentation or something like the Guide documentation. I'll get something added here as a start.

Reply:
Addressed this in 88fb316.